relay-config/src/config.rs

```diff
@@ -643,6 +700,160 @@ impl Config {
         Ok(config)
     }
 
+    /// Override configuration with values coming from other sources (e.g. env variables or
+    /// command line parameters)
+    pub fn override_config(
```

Comment:
_nit_: This function is defined on `Config`, so the `_config` suffix is redundant.

```suggestion
    pub fn override(
```

Reply:
`override` is a reserved keyword in Rust. changed to apply_override